src/ImageSharp/ColorSpaces/CieXyChromaticityCoordinates.cs

```diff
@@ -2,22 +2,21 @@
 // Licensed under the Apache License, Version 2.0.
 
 using System;
-using System.ComponentModel;
-using System.Numerics;
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 
 // ReSharper disable CompareOfFloatsByEqualityOperator
 namespace SixLabors.ImageSharp.ColorSpaces
 {
     /// <summary>
-    /// Represents the coordinates of CIEXY chromaticity space
+    /// Represents the coordinates of CIEXY chromaticity space.
     /// </summary>
-    internal readonly struct CieXyChromaticityCoordinates : IEquatable<CieXyChromaticityCoordinates>, IAlmostEquatable<CieXyChromaticityCoordinates, float>
+    [StructLayout(LayoutKind.Sequential)]
+    internal readonly struct CieXyChromaticityCoordinates
+        : IEquatable<CieXyChromaticityCoordinates>, IAlmostEquatable<CieXyChromaticityCoordinates, float>
     {
-        /// <summary>
-        /// The backing vector for SIMD support.
-        /// </summary>
-        private readonly Vector2 backingVector;
+        // NOTE: We don't implement a backing vector on this class to avoid runtime bugs.
```

Comment:
Way ahead of you 😉 

https://github.com/SixLabors/ImageSharp/commit/1223f786240d05cd7c2abd80565389a7b747dd1e#diff-fdada74c86b339c6ed2bc530c7c8b390R13

Reply:
Hah! I was going remove the backing vector from the other colorspaces next! Nice to know that we can eek even more performance out of this! 

Are we going to try to get your color-transforms branch back in sync or should I carry forward on these refactorings (referencing your branch) ?

Reply:
Reimplemented. Was there anything else other than clean up & bulk conversions in this branch we want to bring over?